iotfunctions/db.py

```diff
@@ -398,22 +380,6 @@ def __init__(self, credentials=None, start_session=False, echo=False, tenant_id=
                                          'DATABASE=xxx;HOSTNAME=xxx;PORT=xxx;UID=xxx;PWD=xxx[;SECURITY=xxx]' % connection_string_from_env)
                     self.db_type = 'db2'
 
-                elif db_type_from_env == 'POSTGRESQL':
-                    sqlalchemy_connection_string = 'postgresql+psycopg2://' + connection_string_from_env
-                    native_connection_string = connection_string_from_env
-                    try:
-                        # Split connection string according to user:password@hostname:port/database
-                        first, last = connection_string_from_env.split("@")
-                        uid, pwd = first.split(":", 1)
-                        hostname_port, database = last.split("/", 1)
-                        hostname, port = hostname_port.split(":", 1)
-                        self.credentials['postgresql'] = {"username": uid, "password": pwd, "db": database,
-                                                          "port": port, "host": hostname}
-                    except Exception:
-                        raise ValueError('Connection string \'%s\' is incorrect. Expected format for POSTGRESQL is '
-                                         'user:password@hostname:port/database' % connection_string_from_env)
-                    self.db_type = 'postgresql'
-
                 else:
                     raise ValueError(
                         'The database type \'%s\' is unknown. Supported types are DB2 and POSTGRESQL' % db_type_from_env)
```

Comment:
Remove the "and POSTGRESQL" from the exception message in this line and in the line further down by 3 lines